Docs/docs/worlds/components/vrc_scenedescriptor.md

```diff
@@ -32,17 +32,18 @@ Describes your VRChat world. Required for every Unity scene you'd like to use as
 | Unity Version                | Unity version being used; you should never need to touch this.                                                  |
 | Dynamic Prefabs              | An array of prefabs that can be dynamically spawned at runtime. Any prefab that isn't referenced in the scene but needs to be spawned should be listed here. |
 | Dynamic Materials            | An array of materials that can be dynamically changed at runtime. Any material that isn't referenced in the scene but needs to be set to an object should be listed here. |
+| Interact Passthrough         | Allow interactions to pass through specified user layers, see description below. |
+
+<!-- Description of Physics Matrix goes here -->
+### Interaction Block and Passthrough on VRChat Layers
 
-<br/>
-<!-- Description of Physics Matrix here -->
-<h3>Interaction Block and Passthrough on VRChat Layers</h3>
 Interaction (grabbing an item from a distance, toggling a UI element with the laser) is blocked by most VRChat layers. The following layers are transparent to interaction and allow you to interact through them:
-<ul>
-<li>UiMenu</li>
-<li>UI</li>
-<li>PlayerLocal</li>
-<li>MirrorReflection</li>
-</ul>
+ - UiMenu
+ - UI
+ - PlayerLocal
+ - MirrorReflection
+   
 <!-- Unreleased feature -->
```

Comment:
Did you mean to leave this in?

Reply:
Yes, but i'm not sure the process here, if we are going to merge this directly, or if its meant to be a reference to someone who is maintaining the final source. It's related to your question to Fax. Should I have two different PRs?